src/lib/components/chat/ChatMessage.svelte

```diff
@@ -195,6 +196,10 @@
 				>
 					<CarbonThumbsDown class="h-[1.14em] w-[1.14em]" />
 				</button>
+				<CopyToClipBoardBtn
```

Comment:
Good idea! There are some adjustments to be made to make it look exactly like the other buttons. I can do it in your PR if you want :)
<img width="435" alt="image" src="https://github.com/huggingface/chat-ui/assets/3841370/3453cdc1-48f0-42d3-87f3-b08fcae5be5f">

Reply:
> We may need to change CopyToClipBoardBtn to fix the padding or to introduce tailwindMerge. I don't know if it's worth it for a such little features

Have you tried overwriting the classes you pass with the [important modifier](https://tailwindcss.com/docs/configuration#important-modifier)? I think it could work.

Reply:
Yes this sould definively work. I didn't know this tailwind feature. Sure if you want to edid go on ;). Otherwise I may do it later in the evening (3 hours max I think)

Reply:
Zero pressure 👍 as you want